config/locales/unsubscriptions.yml

```diff
@@ -0,0 +1,9 @@
+en:
+  unsubscriptions:
+    title:
+      confirm_already_unsubscribed: You’ve already unsubscribed
+      confirm: Are you sure you want to unsubscribe?
+      confirmed: You’ve successfully unsubscribed
+    confirmation:
+      with_title: You won’t get any more updates about %{title}.
```

Comment:
Interesting - I thought you had to put quotes around translations with `%`, but apparently it works fine.